contributing/standards/markdown.md

```diff
@@ -89,6 +89,32 @@ def sample_func(argument1):
 
 A quick way to distinguish between the two cases when it's unclear - if this is code the student can run in a terminal, format it that way. If it's code that Exercism might run (in tests, library code the student will write, etc), default to formatting it as runnable code.
 
+## Special blocks (sometimes called admonitions)
+
+We support special types of blocks that can be added to documents to pull out commentary that doesn't fit with the main body of the text. 
+They are similar to this examples seen in the Julia docs:
+
+<img width="500" alt="Screenshot 2021-03-13 at 17 15 04" src="https://user-images.githubusercontent.com/286476/111038207-aca0bd00-841f-11eb-95fb-20a93943d3dd.png">
+
+We support three types of blocks:
+- **note:** Blocks that pull out some extra special information
+- **caution:** Things that people should know about or tread carefully with
+- **advanced:** Information that is only relevant for people who want to dig more deeply into something or are expected to have more advanced knowledge.
```

Comment:
In a previous example on Slack these were prefixed with `exercism/note` iirc. Which syntax is it?

Reply:
Ah yeah - that changed, thanks. It's with `exercism/`

Reply:
Updated.